lua/entities/gmod_wire_expression2/core/custom/wiring.lua

```diff
@@ -108,7 +108,17 @@ __e2setcost(5)
 e2function wirelink entity:wirelink()
 	if not IsValid(this) then return self:throw("Invalid entity!", nil) end
 	if not isOwner(self, this) then return self:throw("You do not own this entity!", nil) end
-	
+
+	return this
+end
+
+e2function wirelink entity:createWirelink()
```

Comment:
Agreed, this doesn't fix #2940, please revert the behavior of `e:wirelink()`

Reply:
I literally tested it, it's fixed in this PR.

Reply:
Nevermind, carry on